### PR TITLE
[Identity] Butter M1 UX tweak

### PR DESCRIPTION
**Pull request by ccen-stripe**
## Summary

* consent page
	* icon size to 20pt, padding to 16pt
	* title font - medium to bold
	* plus icon replaced by ellipsis icon
	* blullet line color (same on both light/dark)
	* link color
* bottomsheet
	* subtitle color, content color, content line spacing
	* fixed bullet points
* docwarmup
	* title font - medium to bold
	* removed instructions box, instead built a string
		* note we didn't add the "and" there, since this would make translations hard, as we are hardcoding "accepted forms of ID" from client, but getting the actual ID names from server.
	* moved instructions to top of button.

Android: https://github.com/stripe/stripe-android/pull/7638

## Motivation

Address UX feedbacks

## Testing

screenshot tests

## Recording
| Before  | After |
| ------------- | ------------- |
| before  | after |

### CODE REVIEW

**Comment by ccen-stripe** [comment]
@stripe/stripe-identity-observers

File: StripeIdentity/StripeIdentity/Resources/Localizations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -42,6 +42,9 @@
 /* Message for invalid Date of birth field */
 "Date of birth does not look valid" = "Date of birth does not look valid";
 
+/* dirver's license */
```
Comment (wooj-stripe) [inline StripeIdentity/StripeIdentity/Resources/Localizations/en.lproj/Localizable.strings:45]: Seems like a typo: dirver's -> driver's

File: StripeIdentity/StripeIdentity/Source/NativeComponents/Views/DocumentWarmupView.swift [hunk 2]
```diff
@@ -51,63 +49,49 @@ class DocumentWarmupView: UIView {
...
+        bindAcceptTypesOfId(allowList: Array(staticContent.idDocumentTypeAllowlist.keys.map { value in
+            if value == "driving_license" {
+                return String.Localized.driverLicense
+            } else if value == "id_card" {
+                return String.Localized.governmentIssuedId
+            } else if value == "passport" {
+                return String.Localized.passport
+            } else {
+                return ""
+            }
```
Comment (wooj-stripe) [inline StripeIdentity/StripeIdentity/Source/NativeComponents/Views/DocumentWarmupView.swift:74]: If we do return "", seems like these need to be filtered out?

File: StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityFlowView.swift [hunk 3]
```diff
@@ -231,12 +231,12 @@ extension IdentityFlowView {
...
     fileprivate func installConstraints() {
         buttonBackgroundView.translatesAutoresizingMaskIntoConstraints = false
-        privacyPolicyView.translatesAutoresizingMaskIntoConstraints = false
+        buttonTopContentView.translatesAutoresizingMaskIntoConstraints = false
 
         NSLayoutConstraint.activate([
-            // Constrain privacy and policies to top of bottons
-            privacyPolicyView.centerXAnchor.constraint(equalTo: centerXAnchor),
-            privacyPolicyView.bottomAnchor.constraint(equalTo: buttonBackgroundView.topAnchor, constant: -Style.privacyPolicyBottomPadding),
+            // Constrain buttonTop top of bottons
```
Comment (wooj-stripe) [inline StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityFlowView.swift:237]: s/bottons/buttons

File: StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift [hunk 4]
```diff
@@ -66,13 +66,53 @@ final class HTMLTextView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // Check if htmlText is <ul></ul>, if so, return the <li> within, return nil otherwise
+    fileprivate func tryParseUl(htmlString: String) -> [String]? {
```
Comment (wooj-stripe) [inline StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift:70]: is it possible to add tests for this?

File: StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift [hunk 5]
```diff
@@ -98,10 +137,7 @@ final class HTMLTextView: UIView {
         DispatchQueue.main.async { [weak self] in
             do {
                 // Recompute attributed text with updated font sizes
```
Comment (wooj-stripe) [inline StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift:139]: this comment now seems out of date?

File: StripeIdentity/StripeIdentity/Source/NativeComponents/Views/DocumentWarmupView.swift (hunk 2 above)
Comment (ccen-stripe) [reply StripeIdentity/StripeIdentity/Source/NativeComponents/Views/DocumentWarmupView.swift:74]: thanks for the pointer! just replaced with `compatMap`

File: StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift (hunk 5 above)
Comment (ccen-stripe) [reply StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift:139]: It's actually still doing the samething inside `configure`, updated the comment a bit

File: StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift (hunk 4 above)
Comment (ccen-stripe) [reply StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift:70]: thanks for catching this, this indeed breaks something - it matches partially within the html text, so if there's something before or after `<ul></ul>` they'll all be ignored. 
Just updated the logic and corresponding test

**Review by wooj-stripe — APPROVED** [review]
lgtm!

**Pull request merged**